application.conf

```diff
@@ -38,9 +39,12 @@ dm {
   analysesPath="/analyses"
   analysesResolvePath="/analyses/%s"
   analysesUpdatePath="/analyses/%s/outputs"
+  ubamCollectionPath="/collections"
```

Comment:
These should be versioned, matching up with https://github.com/broadinstitute/vault-datamanagement/pull/28

I believe they should be `/collections/v1` based on looking at the other repo.